resources/public/contracts/src/NameBazaarRegistrar.sol

```diff
@@ -18,13 +23,7 @@ contract NameBazaarRegistrar is HashRegistrar {
      * @param _hash The sha3 hash of the label to register.
      */
     function register(bytes32 _hash) public payable {
-        Deed bid = (new DeedImplementation).value(msg.value)(msg.sender);
-        sealedBids[msg.sender][bytes32(0)] = bid;
-        Entry storage newAuction = _entries[_hash];
-        newAuction.registrationDate = now - 6 days;
-        newAuction.value = msg.value;
-        newAuction.highestBid = msg.value;
-        newAuction.deed = bid;
-        trySetSubnodeOwner(_hash, msg.sender);
+        controllers[msg.sender] = true;
```

Comment:
This way there will be no `ControllerAdded` event. I suggest stating that in a comment explicitly.

It might bite us in the future if we will be debugging events.

Reply:
I added the emit.